dotnet/src/Microsoft.Agents.AI.AGUI/Shared/AGUIEventTypes.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+#if ASPNETCORE
+namespace Microsoft.Agents.AI.Hosting.AGUI.AspNetCore.Shared;
```

Comment:
why do we choose namespace here?

Reply:
Both `Microsoft.Agents.AI.Hosting.AGUI.AspNetCore` and `Microsoft.Agents.AI.AGUI` share all the types within the `Shared` folder. Since we give `InternalsVisibleTo` to the tests projects, we run into issues when we reference one of the shared types, because the compiler sees two definitions for the same type coming from two separate assemblies.

The solution is to put them in a namespace specific to that project. We chose to it this way because we didn't want to create a public AGUI library at the moment, but that might change in the future. If we choose to do so, then we would have a set of public types we will reuse.